translations/bg.json

```diff
@@ -0,0 +1,12 @@
+{
+    "panel": {
+        "config": "",
+        "states": "Състояние",
+        "map": "Карта",
+        "logbook": "Дневник",
+        "history": "История",
+        "log_out": "Излез",
+        "mailbox": "",
```

Comment:
How does our translation engine handle this? Will they render an empty string instead of using the fallback?

Reply:
Good catch! Looks like it's sneaking through. I'll update the gulp-merge-json config to ensure blank strings don't replace fallback values.

Reply:
This empty string is still present in git?

Reply:
For now, that's how it was downloaded from Lokalise. At some point soon I'll probably add some minification/rollup to the download script as well, if for no other reason than to deter people from trying to edit them directly.

Reply:
I meant that we should run the updated script and check in those changes, otherwise these translations are broken right now ?

Reply:
Filtered out by the Lokalise export now, but the rollup changes are probably reasonbale to keep in as a failsafe anyway, in case empty strings or even nulls or something sneak in somehow.